tdmpc2/common/layers.py

```diff
@@ -59,7 +59,7 @@ def __init__(self):
 		super().__init__()
 
 	def forward(self, x):
-		return x.div_(255.).sub_(0.5)
+		return x.div(255.).sub(0.5)
```

Comment:
avoid in-place ops in PyTorch unless absolutely necessary (eg, updating params of a nn.Module).
I believe compile undoes the in-place and contrary to common belief in-place is rarely faster than out-place